### PR TITLE
Updating das extension metering dashboard artifacts

### PR DESCRIPTION
**Pull request by Thanu**
This p/r is to update metering-dashboard artifacts with following changes:
1. Adding authentication logic to metering api calls
2. Removing datasource and filter variables from gadgets
3. Updating license header in dashboard.jag file
4. Updating metering-dashboard README, mysql script and CApp pom files

### CODE REVIEW

File: extensions/das/modules/artifacts/metering-dashboard/capps/pom.xml [hunk 1]
```diff
@@ -23,16 +23,16 @@
...
+        <artifactId>stratos-das-extension</artifactId>
         <version>4.1.5-SNAPSHOT</version>
         <relativePath>../../../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>metering-service-capps</artifactId>
-    <name>Apache Stratos - Metering Service Composite Applications</name>
-    <description>Apache Stratos Metering Service Composite Applications</description>
+    <artifactId>org.apache.stratos.metering.service.capps</artifactId>
```
Comment (ravihansa3000) [inline extensions/das/modules/artifacts/metering-dashboard/capps/pom.xml]: why was this artifactId changed?

Comment (Thanu) [inline extensions/das/modules/artifacts/metering-dashboard/capps/pom.xml]: Changed it to follow the same pattern as in other artifactIds in Stratos.

Comment (ravihansa3000) [inline extensions/das/modules/artifacts/metering-dashboard/capps/pom.xml]: Usually this pattern is followed if there are Java sources in the modules. Do we have any Java sources inside CApps?

Comment (Thanu) [inline extensions/das/modules/artifacts/metering-dashboard/capps/pom.xml]: No, there is no java file.

Comment (Thanu) [inline extensions/das/modules/artifacts/metering-dashboard/capps/pom.xml]: Fixed in commit 57d497dc4f419e004a83928785e761fb31c97b87.

**Pull request merged**